aten/src/THCS/THCSparse.cu

```diff
@@ -1,167 +1,41 @@
 #include "THCSparse.h"
 
 void THCudaSparse_Xcoo2csr(THCState *state, const int *coorowind, int64_t nnz, int64_t m, int *csrrowptr) {
-  THAssertMsg((m <= INT_MAX) && (nnz <= INT_MAX),
-    "cusparseXcoo2csr only supports m, nnz with the bound [val] <= %d",
-    INT_MAX);
-  cusparseHandle_t handle = THCState_getCurrentSparseHandle(state);
-  cusparseSetStream(handle, THCState_getCurrentStream(state));
-  THCusparseCheck(cusparseXcoo2csr(handle, coorowind, nnz, m, csrrowptr,
-    TH_INDEX_BASE ? CUSPARSE_INDEX_BASE_ONE : CUSPARSE_INDEX_BASE_ZERO
-  ));
-}
-
-cusparseOperation_t convertTransToCusparseOperation(char trans) {
-  if (trans == 't') return CUSPARSE_OPERATION_TRANSPOSE;
-  else if (trans == 'n') return CUSPARSE_OPERATION_NON_TRANSPOSE;
-  else if (trans == 'c') return CUSPARSE_OPERATION_CONJUGATE_TRANSPOSE;
-  else {
-    THError("trans must be one of: t, n, c");
-    return CUSPARSE_OPERATION_TRANSPOSE;
-  }
-}
-
-void adjustLd(char transb, int64_t m, int64_t n, int64_t k, int64_t *ldb, int64_t *ldc)
-{
-  int transb_ = ((transb == 't') || (transb == 'T'));
-
-  if(n == 1)
-    *ldc = m;
-
-  if(transb_)
-  {
-    if(k == 1)
-      *ldb = n;
-  }
-  else
-  {
-    if(n == 1)
-      *ldb = k;
-  }
+  THError("Internal error! This API is deprecated. Shout if you need it.");
```

Comment:
can we remove these?  It's not hard to add them back and it will really tell you if someone is using them.